src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -172,8 +171,11 @@ public PluginCompatReport testPlugins()
         // Providing XSL Stylesheet along xml report file
         if(config.reportFile != null){
             if(config.isProvideXslReport()){
+                Files.createDirectories(Paths.get(PluginCompatReport.getBaseFilepath(config.reportFile)));
```

Comment:
Some minor cleanup while I was here: reducing use of Plexus `FileUtils` in favor of Java Platform functionality. This again simplifies our dependency tree. But in order to use `Files.copy` a few lines below I needed to first make sure the parent directory exists.